riscv/src/coprocessors.rs

```diff
@@ -270,12 +214,9 @@ pub fn call_every_submachine(coprocessors: &CoProcessors) -> Vec<String> {
     // automatically.
     // https://github.com/powdr-labs/powdr/issues/548
     let mut calls = vec![];
-    if coprocessors.has(BINARY_COPROCESSOR.name) {
-        calls.push("x10 <== and(x10, x10);".to_string());
-    }
-    if coprocessors.has(SHIFT_COPROCESSOR.name) {
-        calls.push("x10 <== shl(x10, x10);".to_string());
-    }
+    calls.push("x10 <== and(x10, x10);".to_string());
+    calls.push("x10 <== shl(x10, x10);".to_string());
```

Comment:
since these moved to the compiler, maybe these also should, but this is quite hacky already and will probably change with following changes, so maybe fine as is